src/components/forms/SignIn.tsx

```diff
@@ -8,16 +8,40 @@ import logo from '../../../public/images/logos/Crowd_Comic_Logo_BW.svg';
 
 import { useState } from 'react';
 import { loginAction } from '@/app/login/actions';
-
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+import Row from 'react-bootstrap/Row';
 
 export function SignInForm() {
     const [message, errorState] = useState('');
     const [passwordVisible, setPasswordVisibility] = useState(false);
+    const [emailValid, setEmailValid] = useState(false);
+    const [passwordValid, setPasswordValid] = useState(false);
+    const [validated, setValidated] = useState(false)
 
     const togglePasswordVisibility = () => {
         setPasswordVisibility(!passwordVisible);
     };
 
+    const handleSubmit = (event: any) => {
+        if (!emailValid || !passwordValid) {
+            event.preventDefault();
+            event.stopPropagation();
+        }
+        setValidated(true);
```

Comment:
After submitting once, validated never gets set to false so each input field's error message will always update on change. Is this the intended function of the validation after a submit click?

Reply:
on the sign in page yes. Validated is meant to keep track if they hit submit the first time to then display the issues.

Reply:
Cool 👍